tests/test_verify_data_hash.py

```diff
@@ -0,0 +1,278 @@
+#! /usr/bin/env python3
```

Comment:
Better to add this feature in one of the existing clients (echo or generic client) and call it from run_tests.sh instead of duplicating the code

Reply:
Done